### ISSUE TITLE
Add some reasonable ESLint rules

### ISSUE THREAD

**Issue by Mr0grog** [5calls/5calls#227]
Now that ESLint is part of the test process, we should actually add some reasonable code style rules to it. (At current, it’s only used to ensure people don’t leave `console` methods in their at PR time).

**Comment by cdoremus** [comment]
+1 @Mr0grog. Could you suggest an ```.eslintrc``` file to start with (or set of rules that we can add to an ```eslintConfig``` field in ```package.json```)?

**Comment by Mr0grog** [comment]
I think someone else who has actually contributed substantively to the JS codebase should choose a set of rules that best match the intended coding style here. I’m pretty sure I’m not that person; I haven’t examined much of the JS source in any detail and only made two very minimum-impact changes.

**Comment by oderby** [comment]
As someone who's contributed a little bit more, but not much, I think it's safe to say there's not a lot of opinionated people around here that will object overly much to a sane recommendation. So, going off that...any reason not to just use the [recommended set of rules](http://eslint.org/docs/user-guide/configuring#using-eslintrecommended)? Turning those on and running on our code base only produces two types of errors: ["no-undef"](http://eslint.org/docs/rules/no-undef) and ["no-unused-vars"](http://eslint.org/docs/rules/no-unused-vars), both of which seem easy to fix/turn into warnings/turn off.

**Comment by cdoremus** [comment]
@oderby based on your recommendations, I modified the .eslitrc.json file in the static folder to look like this:
```
{
  "parserOptions": {
    "ecmaVersion": 7,
    "sourceType": "module"
  },
  "env": {
    "browser": true,
    "commonjs": true,
    "mocha": true
  },
  "extends": "eslint:recommended",
  "rules": {
      "no-unused-vars": "warn",
      "no-undef":  "warn",
      "no-console": "error"
  }
}
```
I set the ```no-unused-vars``` and ```no-undef``` rules to issue a warning rather than being ignored (value === "off") because these things should be cleaned up as time allows.

Please comment.

**Comment by nickoneill** [comment]
The rules specified here look totally reasonable. We can add/improve them as issues appear but this  is an excellent place to start.

**Comment by cdoremus** [comment]
@nickoneill thanks for chiming in. I've assigned this to myself and will put together a PR in a day or two to give anyone else a chance to comment on my proposed changes.

**Comment by cdoremus** [comment]
This issue has been implemented in PR #233
